Mods/ArchipelagoCore.SC2Mod/Base.SC2Data/Lib5BD4895D_h.galaxy

```diff
@@ -64,7 +73,7 @@ struct lib5BD4895D_gs_APObjectiveMainPanel {
 };
 
 // Variable Declarations
-bool[lib5BD4895D_gv_aP_Core_LOCATIONS_PER_MAP + 1] lib5BD4895D_gv_aP_Core_locationCollectStatus;
+bool[lib5BD4895D_gv_aP_Core_LOCATIONS_PER_MAP + 1] lib5BD4895D_gv_aP_Core_locationCollectable;

```

Comment:
Just noting that I renamed the variable to be more clear (as "CollectStatus" = True could be interpretted as _either_ "can be collected" or "has been collected").